### PR TITLE
Make it possible to pass arguments to cvxopt solver

### PR DESCRIPTION
**Pull request by whoburg**
Fixed a bug and added a test. See inline comments for more details.

### CODE REVIEW

File: gpkit/_cvxopt.py [hunk 1]
```diff
@@ -2,10 +2,8 @@
 from cvxopt import solvers, spmatrix, matrix, log
 
 
-def cvxoptimize_fn(options=None):
-    "Return a cvxopt solve function for a particular set of options."
-    if options:
-        solvers.options.update(options)
```
Comment (whoburg) [inline gpkit/_cvxopt.py]: this update of `solvers.options` has non-intuitive side effects, so I've removed it. For example, if one runs

`m.solve(solver="cvxopt", options={"feastol": 1e-6})`

and then runs

`m.solve(solver="cvxopt")`,

the second call will still have the feastol option from the earlier call.

File: gpkit/geometric_program.py [hunk 2]
```diff
@@ -152,7 +152,7 @@ def _get_solver(solver):
 
             if solver == "cvxopt":
                 from ._cvxopt import cvxoptimize_fn
-                solverfn = cvxoptimize_fn(*args, **kwargs)
+                solverfn = cvxoptimize_fn()
```
Comment (whoburg) [inline gpkit/geometric_program.py]: `cvxoptimize_fn` did not accept args or kwargs, so this line fails if a user tries to specify args or kwargs.

File: gpkit/tests/t_model.py [hunk 3]
```diff
@@ -390,10 +390,22 @@ def test_partial_sub_signomial(self):
...
+class TestModelSolverSpecific(unittest.TestCase):
+    """test cases run only for specific solvers"""
+    def test_cvxopt_kwargs(self):
+        if "cvxopt" not in settings["installed_solvers"]:
+            return
+        x = Variable("x")
+        m = Model(x, [x >= 12])
+        # make sure it's possible to pass the kktsolver option to cvxopt
+        sol = m.solve(solver="cvxopt", verbosity=0, kktsolver="ldl")
+        self.assertAlmostEqual(sol["cost"], 12., NDIGS["cvxopt"])
```
Comment (whoburg) [inline gpkit/tests/t_model.py]: here's a place to put tests in `t_model` that shouldn't get run for every solver.

**Comment by whoburg** [comment]
I'm going to merge this. It passes tests and is needed on a project.

@bqpd, can we please do a minor pip release to capture the recent bug fixes?

**Comment by bqpd** [comment]
For sure!

**Pull request merged**